python/foglamp/common/service_record.py

```diff
@@ -25,11 +25,23 @@ class Type(IntEnum):
         Core = 2
         Southbound = 3
 
+    class Status(IntEnum):
+        """Enumeration for Service Status"""
+
+        Running = 1
+        Unregistered = 2
```

Comment:
what is it? why an unregistered will be in Service Registry

Reply:
Right. "Unregistered" changed to "Stopped".